mkosi/_version.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import datetime
+import os
+import subprocess
+from importlib.metadata import PackageNotFoundError, version
+from pathlib import Path
+
+try:
+    __version__ = version("package-name")
+except PackageNotFoundError:
+    # This branch means, that mkosi was not installed as a package as Python is aware, this is most likely
+    # the case if mkosi was run via the bin/mkosi shim in mkosi's repository. Though discouraged, we fall
+    # back to importing setuptools_scm directly and having it try figure stuff out.
```

Comment:
You are not importing anything here?

Reply:
That comment was leftover from the first version that called setuptools-scm directly, but I wanted to have the fallback work without having having to have that installed.